Makefile

```diff
@@ -1,5 +1,6 @@
+CC = clang
 LDLIBS = -lcurses
-CFLAGS = -g
+CFLAGS = -g -Wall
```

Comment:
These options have to move to **configure.ac** after auto* tools merge.

Reply:
We're deleting all of Makefile, anyway, right?

Reply:
yes. Need to bring these into configure.ac